python/private/toolchains_repo.bzl

```diff
@@ -64,10 +65,15 @@ py_toolchain_suite(
     user_repository_name = "{user_repository_name}_{platform}",
     prefix = "{prefix}{platform}",
     target_compatible_with = {compatible_with},
+    flag_values = {flag_values},
     python_version = "{python_version}",
     set_python_version_constraint = "{set_python_version_constraint}",
 )""".format(
-            compatible_with = meta.compatible_with,
+            compatible_with = render.indent(render.list(meta.compatible_with)).lstrip(),
+            flag_values = render.indent(render.dict(
```

Comment:
This is breaking the Windows build due to some reason, so we may want to revert this commit and split it into a separate PR.

Reply:
Found the issue and fixed in https://github.com/bazelbuild/rules_python/pull/1837/commits/8a6bda59e95466c4922ca7c32392200d5c19c008